#! /bin/sh /usr/share/dpatch/dpatch-run
## 23_fix_faq_id.dpatch by Roland Rosenfeld <roland@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix value of id attribute in <sect2> tag (no underscore allowed).

@DPATCH@
diff -urNad privoxy~/doc/source/faq.sgml privoxy/doc/source/faq.sgml
--- privoxy~/doc/source/faq.sgml
+++ privoxy/doc/source/faq.sgml
@@ -149,7 +149,7 @@
 <!--   ~~~~~       New section      ~~~~~     -->
 
 <sect1 id="general"><title>General Information</title>
-<sect2 renderas="sect3" id="who_uses"><title>Who should use Privoxy?</title>
+<sect2 renderas="sect3" id="whouses"><title>Who should use Privoxy?</title>
  <para>
   Anyone that is interested in security, privacy, or in 
   finer-grained control over their web and Internet experience.
